test/functional/test-action.js

```diff
@@ -544,6 +663,64 @@ describe('Action method', () => {
 });
 
 
+describe('Multiple handlers action method', () => {
+  let sandbox;
+  let win;
+  let action;
+  let onEnqueue1, onEnqueue2;
+  let targetElement, parent, child, execElement1, execElement2;
+
+  function createExecElement(id, enqueAction) {
+    const execElement = document.createElement('amp-element');
+    execElement.setAttribute('id', id);
+    execElement.enqueAction = enqueAction;
+    return execElement;
+  }
+
+  beforeEach(() => {
+    sandbox = sinon.sandbox.create();
+    win = {
+      document: {body: {}},
+      services: {
+        vsync: {obj: {}},
+      },
+    };
+    action = new ActionService(new AmpDocSingle(win), document);
+    onEnqueue1 = sandbox.spy();
+    onEnqueue2 = sandbox.spy();
+    targetElement = document.createElement('target');
+    const id1 = ('Ea' + Math.random()).replace('.', '');
+    const id2 = ('Eb' + Math.random()).replace('.', '');
+    targetElement.setAttribute('on', `tap:${id1}.method1,${id2}.method2`);
+    parent = document.createElement('parent');
+    child = document.createElement('child');
+    parent.appendChild(targetElement);
+    targetElement.appendChild(child);
+    document.body.appendChild(parent);
+
+    execElement1 = createExecElement(id1, onEnqueue1);
+    execElement2 = createExecElement(id2, onEnqueue2);
+
+    parent.appendChild(execElement1);
+    parent.appendChild(execElement2);
```

Comment:
Ah, I see this was copied from `describe('Action method')`. We should extract into a top-level helper method.